staging/src/k8s.io/kubectl/pkg/cmd/apply/apply_test.go

```diff
@@ -788,28 +805,53 @@ func TestApplyPruneObjectsWithAllowlist(t *testing.T) {
 			},
 		},
 	}
-	if err != nil {
-		t.Fatal(err)
-	}
 
 	testCases := map[string]struct {
 		currentResources        []runtime.Object
 		pruneAllowlist          []string
+		namespace               string
 		expectedPrunedResources []string
 		expectedOutputs         []string
 	}{
-		"prune without allowlist should delete resources that are not in the specified file": {
-			currentResources:        []runtime.Object{rc, rc2, cm},
-			expectedPrunedResources: []string{"test/test-cm", "test/test-rc2"},
+		// Deprecated: kubectl apply will no longer prune non-namespaced resources by default when used with the --namespace flag in a future release
+		// namespace is a non-namespaced resource and will not be pruned in the future
```

Comment:
Doesn't this note belong on the second test, i.e. isn't that the case that we'll need to update later?

Reply:
Yes, Updated.

It should be with a test case with namespace specified but without pruneAllowlist.